lib/generators/capistrano/sidekiq/systemd/templates/sidekiq.service.capistrano.erb

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=sidekiq for <%= "#{fetch(:application)} (#{fetch(:stage)})" %>
+After=syslog.target network.target
+
+[Service]
+Type=simple
+Environment=RAILS_ENV=<%= fetch(:rails_env) %>
+WorkingDirectory=<%= fetch(:deploy_to) %>/current
+ExecStart=<%= fetch(:bundler_path, '/usr/local/bin/bundler') %> exec sidekiq -e <%= fetch(:rails_env) %>
```

Comment:
I'm not an expert on systemd, but it seems that ExecStart is *not*  taking into consideration sidekiq.yml config file, right? Since this is oficially supported by the gem via the `set :sidekiq_config` option (see https://github.com/seuros/capistrano-sidekiq/wiki), I think it should be supported here as well.

Reply:
What do you mean? This just call `bundle exec sidekiq` via systemd, it should be some sort of an equivalent to `cap sidekiq:start`. Nothing less, nothing more.

I have an idea to add a generator for this file, which can solve some issue. In the mean time, this is just a template for a service file and you can customize the resulting systemd service file to your liking.